tests/test_notifications.py

```diff
@@ -1,19 +1,20 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""Test files for notifications API."""
 
 import pytest
 from selenium.common.exceptions import TimeoutException
 
 from foxpuppet.windows.browser.notifications import BaseNotification
-from foxpuppet.windows.browser.notifications.addons import (
-    AddOnInstallBlocked,
-    AddOnInstallComplete,
-    AddOnInstallConfirmation)
+from foxpuppet.windows.browser.notifications.addons import (AddOnInstallBlocked, # noqa
+                                                            AddOnInstallComplete, # noqa
+                                                            AddOnInstallConfirmation) # noqa
```

Comment:
I think we should skip the isort warning, rather than all warnings. I'll take care of this in a followup.